### PR TITLE
Docs(web-react, web-twig, web): Add Tooltip demo #DS-914

### PR DESCRIPTION
**Pull request by pavelklibani**
### Additional context

Fixed bug in web & web-twig FloatingUi demo _(data-placement => data-spirit-placement)_

Updated version of FloatingUI from 0.5.4 to 1.5.3 in .mjs file 
(packages/web/src/scss/components/Tooltip/floating-ui-example.mjs)

### Issue reference

[Demo: Tooltip má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-914)

### CODE REVIEW

**Comment by literat** [comment]
> Should we keep `Dismissible Tooltip via JS API` in web demo?

Yes, please.

File: packages/web-react/src/components/Tooltip/demo/TooltipFloatingUi.tsx [hunk 1]
```diff
@@ -38,8 +35,14 @@ const Story: ComponentStory<typeof Tooltip> = () => {
         The following example is using external library <a href="https://floating-ui.com">Floating UI</a>.
       </p>
       <p>🖱 Try scrolling the example to see how Tooltip placement is updated.</p>
-      <div className="docs-FloatingUI-Wrapper">
-        <div className="docs-FloatingUI-Container">
+      <div
+        className="docs-FloatingUI-Wrapper  bg-cover"
```
Comment (crishpeen) [inline packages/web-react/src/components/Tooltip/demo/TooltipFloatingUi.tsx:39]:
```suggestion
        className="docs-FloatingUI-Wrapper bg-cover"
```

File: packages/web-twig/src/Resources/components/Tooltip/stories/TooltipFloatingUI.twig [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        top: `${y}px`,
+        left: `${x}px`,
+      });
+      tooltip.dataset.spiritPlacement = placement;
+    });
+  });
+
+  // Call cleanup function when tooltip is removed from DOM.
+  // cleanup();
+</script>
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Tooltip/stories/TooltipFloatingUI.twig:64]: missing an empty line at the bottom of the file? @literat config problem?

File: packages/web-twig/src/Resources/components/Tooltip/stories/TooltipDismissibleViaJS.twig [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+</p>
+
+<TooltipWrapper>
+  <Button id="my-dismissible-button" aria-describedby="my-dismissible-tooltip2">I have a tooltip 😎</Button>
+  <Tooltip id="my-dismissible-tooltip2" placement="right" isDismissible UNSAFE_className="is-hidden">
+      Close me
+  </Tooltip>
+</TooltipWrapper>
+
+{{ encore_entry_script_tags('tooltipDismissibleViaJS') }}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Tooltip/stories/TooltipDismissibleViaJS.twig:12]: again an empty line at the bottom of the file, please

**Review by crishpeen — APPROVED** [review]

File: packages/web-twig/src/Resources/components/Tooltip/stories/TooltipDismissibleViaJS.twig (hunk 3 above)
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/Tooltip/stories/TooltipDismissibleViaJS.twig]: space added

File: packages/web-twig/src/Resources/components/Tooltip/stories/TooltipFloatingUI.twig (hunk 2 above)
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/Tooltip/stories/TooltipFloatingUI.twig]: space added

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-twig/src/Resources/components/Tooltip/stories/TooltipFloatingUI.twig (hunk 2 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/Tooltip/stories/TooltipFloatingUI.twig]: @crishpeen IDE should respect `.editorconfig` in the root of this project.

**Pull request merged**